### DISCUSSION TITLE
Packaged Browser Solution

### DISCUSSION THREAD

**Discussion by rogerclarkmelbourne** [airsdk/Adobe-Runtime-Support#600] [Q&A]
We've been trying to contact Harman on multiple occasions about their "Packaged Browser" , however we just got an initial referral and since then have not get any responses from Harman to our many follow-up emails.

We've also tried to directly email @ajwfrost , but have not received a reply.

With the removal of Flash support in most browsers in less than 3 months, we are keen to have a path forward for our content to remain online.

**Comment by marchbold** [comment]
Feel free to contact us (distriqt), we have provided packaged browsers and converted sites into dedicated AIR apps for several clients. You can reach us at airnativeextensions@distriqt.com.

**Comment by rogerclarkmelbourne** [comment]
@marchbold 

Thanks.

I'll pass on your details.

**Comment by navbhandari1990** [comment]
Can you please guide me how do I get harman's package browser solution for my organization to use flash for internal application post 2020 ? Sending email to harman support  is not helping as no response. Please let me know what is the process

**Comment by rogerclarkmelbourne** [comment]
@navbhandari1990 

AFIK. Harman are still not replying to emails about the packaged browser :-(

**Comment by ajwfrost** [comment]
Hi - I've tried looking through our support inbox but couldn't find any emails about this that the marketing team hadn't responded to.. if you fill in the contact form at the bottom of the below web page, it should go more directly to an appropriate marketing contact though:
https://services.harman.com/partners/adobe

**Comment by navbhandari1990** [comment]
@ajwfrost  Thanks for checking , I did recived the response recently and working with Harman support.
